docs/source/developers/deployment.rst

```diff
@@ -29,74 +29,81 @@ Once you have a repository ready:
   # Authenticate to quay.io
   docker login <credentials> quay.io
 
-  # Tag the build 
-  docker tag ibm-spectrum-scale-csi-operator quay.io/<your-user>/ibm-spectrum-scale-csi-operator:v0.9.1
+  # Configure some variables
+  VERSION="v0.9.1"
```

Comment:
We should change to latest available version number 1.0.0

Reply:
We could.. 

The intent here is not to track the current version of the  code (because that's a maintainability nightmare) But since this is building the code locally and tagging it, would having this variable at 0.0.1 be better to disconnect the manual build with our product build version ?